### PR TITLE
[API] creat org repo call same as github

### LINKED ISSUE #9090: API: /org/{org}/repos is inconsistent with the rest of the api
- Gitea version (or commit ref): 1.11.0+dev-213-g173da1b5a
- Git version: 2.22.0
- Operating system: linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes
  - [ ] No
  - [x] Not relevant

## Description

The api for creating a new repo for an organisation uses the /api/v1/org/{org} prefix instead of the /api/v1/orgs/{org} prefix like the rest of the api. 

I have an (untested) change in my fork: https://github.com/ederuiter/gitea which deprecates the /api/v1/org/{org}/repos call and adds a new /api/v1/orgs/{org}/repos call. If this is something you want I can test it and create a pull request for it.

### PR DESCRIPTION
**Pull request by 6543**
close #9090

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

File: routers/api/v1/api.go [hunk 1]
```diff
@@ -593,17 +593,15 @@ func RegisterRoutes(m *macaron.Macaron) {
...
 
 		// Repositories
-		m.Post("/org/:org/repos", reqToken(), bind(api.CreateRepoOption{}), repo.CreateOrgRepo)
+		m.Post("/org/:org/repos", reqToken(), bind(api.CreateRepoOption{}), repo.CreateOrgRepoDeprecated)
+
+		m.Combo("/repositories/:id", reqToken()).Get(repo.GetByID)
 
 		m.Group("/repos", func() {
 			m.Get("/search", repo.Search)
-		})
```
Comment (lunny) [inline routers/api/v1/api.go:611]: Are there the same group middlewares between these two group?

**Review by lunny — CHANGES_REQUESTED** [review]

File: routers/api/v1/api.go (hunk 1 above)
Comment (6543) [reply routers/api/v1/api.go:611]: same end here : [link](https://github.com/go-gitea/gitea/pull/9186/files#diff-2620fdbeeb83ce43692534e6c2c39452R803)

so there is NO reqXY in both of these Groups

Comment (6543) [reply routers/api/v1/api.go:611]: @lunny or do you mean something else?!?

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
ready to merge :rocket:

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**